---a/net/minecraft/init/Blocks.java
+++b/net/minecraft/init/Blocks.java
@@ -34,8 +34,8 @@
 import net.minecraft.block.BlockTripWireHook;
 import net.minecraft.util.ResourceLocation;
 
-public class Blocks
-{
+public class Blocks {
+
     public static final Block air;
     public static final Block stone;
     public static final BlockGrass grass;
@@ -236,41 +236,35 @@
     public static final BlockSlab stone_slab2;
     private static final String __OBFID = "CL_00000204";
 
-    private static Block getRegisteredBlock(String p_180383_0_)
-    {
-        return (Block)Block.blockRegistry.getObject(new ResourceLocation(p_180383_0_));
+    private static Block getRegisteredBlock(String p_180383_0_) {
+        return (Block) Block.blockRegistry.getObject(new ResourceLocation(p_180383_0_));
     }
 
-    static
-    {
-        if (!Bootstrap.isRegistered())
-        {
+    static {
+        if (!Bootstrap.isRegistered()) {
             throw new RuntimeException("Accessed Blocks before Bootstrap!");
-        }
-        else
-        {
+        } else {
             air = getRegisteredBlock("air");
             stone = getRegisteredBlock("stone");
-            grass = (BlockGrass)getRegisteredBlock("grass");
+            grass = (BlockGrass) getRegisteredBlock("grass");
             dirt = getRegisteredBlock("dirt");
             cobblestone = getRegisteredBlock("cobblestone");
             planks = getRegisteredBlock("planks");
             sapling = getRegisteredBlock("sapling");
             bedrock = getRegisteredBlock("bedrock");
-            flowing_water = (BlockDynamicLiquid)getRegisteredBlock("flowing_water");
-            water = (BlockStaticLiquid)getRegisteredBlock("water");
-            flowing_lava = (BlockDynamicLiquid)getRegisteredBlock("flowing_lava");
-            lava = (BlockStaticLiquid)getRegisteredBlock("lava");
-            sand = (BlockSand)getRegisteredBlock("sand");
+            flowing_water = (BlockDynamicLiquid) getRegisteredBlock("flowing_water");
+            water = (BlockStaticLiquid) getRegisteredBlock("water");
+            flowing_lava = (BlockDynamicLiquid) getRegisteredBlock("flowing_lava");
+            lava = (BlockStaticLiquid) getRegisteredBlock("lava");
+            sand = (BlockSand) getRegisteredBlock("sand");
             gravel = getRegisteredBlock("gravel");
             gold_ore = getRegisteredBlock("gold_ore");
             iron_ore = getRegisteredBlock("iron_ore");
             coal_ore = getRegisteredBlock("coal_ore");
             log = getRegisteredBlock("log");
             log2 = getRegisteredBlock("log2");
-            leaves = (BlockLeaves)getRegisteredBlock("leaves");
-            leaves2 = (BlockLeaves)getRegisteredBlock("leaves2");
+            leaves = (BlockLeaves) getRegisteredBlock("leaves");
+            leaves2 = (BlockLeaves) getRegisteredBlock("leaves2");
             sponge = getRegisteredBlock("sponge");
             glass = getRegisteredBlock("glass");
             lapis_ore = getRegisteredBlock("lapis_ore");
@@ -280,32 +280,32 @@
             bed = getRegisteredBlock("bed");
             golden_rail = getRegisteredBlock("golden_rail");
             detector_rail = getRegisteredBlock("detector_rail");
-            sticky_piston = (BlockPistonBase)getRegisteredBlock("sticky_piston");
+            sticky_piston = (BlockPistonBase) getRegisteredBlock("sticky_piston");
             web = getRegisteredBlock("web");
-            tallgrass = (BlockTallGrass)getRegisteredBlock("tallgrass");
-            deadbush = (BlockDeadBush)getRegisteredBlock("deadbush");
-            piston = (BlockPistonBase)getRegisteredBlock("piston");
-            piston_head = (BlockPistonExtension)getRegisteredBlock("piston_head");
+            tallgrass = (BlockTallGrass) getRegisteredBlock("tallgrass");
+            deadbush = (BlockDeadBush) getRegisteredBlock("deadbush");
+            piston = (BlockPistonBase) getRegisteredBlock("piston");
+            piston_head = (BlockPistonExtension) getRegisteredBlock("piston_head");
             wool = getRegisteredBlock("wool");
-            piston_extension = (BlockPistonMoving)getRegisteredBlock("piston_extension");
-            yellow_flower = (BlockFlower)getRegisteredBlock("yellow_flower");
-            red_flower = (BlockFlower)getRegisteredBlock("red_flower");
-            brown_mushroom = (BlockBush)getRegisteredBlock("brown_mushroom");
-            red_mushroom = (BlockBush)getRegisteredBlock("red_mushroom");
+            piston_extension = (BlockPistonMoving) getRegisteredBlock("piston_extension");
+            yellow_flower = (BlockFlower) getRegisteredBlock("yellow_flower");
+            red_flower = (BlockFlower) getRegisteredBlock("red_flower");
+            brown_mushroom = (BlockBush) getRegisteredBlock("brown_mushroom");
+            red_mushroom = (BlockBush) getRegisteredBlock("red_mushroom");
             gold_block = getRegisteredBlock("gold_block");
             iron_block = getRegisteredBlock("iron_block");
-            double_stone_slab = (BlockSlab)getRegisteredBlock("double_stone_slab");
-            stone_slab = (BlockSlab)getRegisteredBlock("stone_slab");
+            double_stone_slab = (BlockSlab) getRegisteredBlock("double_stone_slab");
+            stone_slab = (BlockSlab) getRegisteredBlock("stone_slab");
             brick_block = getRegisteredBlock("brick_block");
             tnt = getRegisteredBlock("tnt");
             bookshelf = getRegisteredBlock("bookshelf");
             mossy_cobblestone = getRegisteredBlock("mossy_cobblestone");
             obsidian = getRegisteredBlock("obsidian");
             torch = getRegisteredBlock("torch");
-            fire = (BlockFire)getRegisteredBlock("fire");
+            fire = (BlockFire) getRegisteredBlock("fire");
             mob_spawner = getRegisteredBlock("mob_spawner");
             oak_stairs = getRegisteredBlock("oak_stairs");
-            chest = (BlockChest)getRegisteredBlock("chest");
-            redstone_wire = (BlockRedstoneWire)getRegisteredBlock("redstone_wire");
+            chest = (BlockChest) getRegisteredBlock("chest");
+            redstone_wire = (BlockRedstoneWire) getRegisteredBlock("redstone_wire");
             diamond_ore = getRegisteredBlock("diamond_ore");
             diamond_block = getRegisteredBlock("diamond_block");
             crafting_table = getRegisteredBlock("crafting_table");
@@ -337,9 +337,9 @@
             snow_layer = getRegisteredBlock("snow_layer");
             ice = getRegisteredBlock("ice");
             snow = getRegisteredBlock("snow");
-            cactus = (BlockCactus)getRegisteredBlock("cactus");
+            cactus = (BlockCactus) getRegisteredBlock("cactus");
             clay = getRegisteredBlock("clay");
-            reeds = (BlockReed)getRegisteredBlock("reeds");
+            reeds = (BlockReed) getRegisteredBlock("reeds");
             jukebox = getRegisteredBlock("jukebox");
             oak_fence = getRegisteredBlock("fence");
             spruce_fence = getRegisteredBlock("spruce_fence");
@@ -351,10 +351,10 @@
             netherrack = getRegisteredBlock("netherrack");
             soul_sand = getRegisteredBlock("soul_sand");
             glowstone = getRegisteredBlock("glowstone");
-            portal = (BlockPortal)getRegisteredBlock("portal");
+            portal = (BlockPortal) getRegisteredBlock("portal");
             lit_pumpkin = getRegisteredBlock("lit_pumpkin");
             cake = getRegisteredBlock("cake");
-            unpowered_repeater = (BlockRedstoneRepeater)getRegisteredBlock("unpowered_repeater");
-            powered_repeater = (BlockRedstoneRepeater)getRegisteredBlock("powered_repeater");
+            unpowered_repeater = (BlockRedstoneRepeater) getRegisteredBlock("unpowered_repeater");
+            powered_repeater = (BlockRedstoneRepeater) getRegisteredBlock("powered_repeater");
             trapdoor = getRegisteredBlock("trapdoor");
             monster_egg = getRegisteredBlock("monster_egg");
             stonebrick = getRegisteredBlock("stonebrick");
@@ -375,7 +375,7 @@
             acacia_fence_gate = getRegisteredBlock("acacia_fence_gate");
             brick_stairs = getRegisteredBlock("brick_stairs");
             stone_brick_stairs = getRegisteredBlock("stone_brick_stairs");
-            mycelium = (BlockMycelium)getRegisteredBlock("mycelium");
+            mycelium = (BlockMycelium) getRegisteredBlock("mycelium");
             waterlily = getRegisteredBlock("waterlily");
             nether_brick = getRegisteredBlock("nether_brick");
             nether_brick_fence = getRegisteredBlock("nether_brick_fence");
@@ -383,44 +383,44 @@
             nether_wart = getRegisteredBlock("nether_wart");
             enchanting_table = getRegisteredBlock("enchanting_table");
             brewing_stand = getRegisteredBlock("brewing_stand");
-            cauldron = (BlockCauldron)getRegisteredBlock("cauldron");
+            cauldron = (BlockCauldron) getRegisteredBlock("cauldron");
             end_portal = getRegisteredBlock("end_portal");
             end_portal_frame = getRegisteredBlock("end_portal_frame");
             end_stone = getRegisteredBlock("end_stone");
             dragon_egg = getRegisteredBlock("dragon_egg");
             redstone_lamp = getRegisteredBlock("redstone_lamp");
             lit_redstone_lamp = getRegisteredBlock("lit_redstone_lamp");
-            double_wooden_slab = (BlockSlab)getRegisteredBlock("double_wooden_slab");
-            wooden_slab = (BlockSlab)getRegisteredBlock("wooden_slab");
+            double_wooden_slab = (BlockSlab) getRegisteredBlock("double_wooden_slab");
+            wooden_slab = (BlockSlab) getRegisteredBlock("wooden_slab");
             cocoa = getRegisteredBlock("cocoa");
             sandstone_stairs = getRegisteredBlock("sandstone_stairs");
             emerald_ore = getRegisteredBlock("emerald_ore");
             ender_chest = getRegisteredBlock("ender_chest");
-            tripwire_hook = (BlockTripWireHook)getRegisteredBlock("tripwire_hook");
+            tripwire_hook = (BlockTripWireHook) getRegisteredBlock("tripwire_hook");
             tripwire = getRegisteredBlock("tripwire");
             emerald_block = getRegisteredBlock("emerald_block");
             spruce_stairs = getRegisteredBlock("spruce_stairs");
             birch_stairs = getRegisteredBlock("birch_stairs");
             jungle_stairs = getRegisteredBlock("jungle_stairs");
             command_block = getRegisteredBlock("command_block");
-            beacon = (BlockBeacon)getRegisteredBlock("beacon");
+            beacon = (BlockBeacon) getRegisteredBlock("beacon");
             cobblestone_wall = getRegisteredBlock("cobblestone_wall");
             flower_pot = getRegisteredBlock("flower_pot");
             carrots = getRegisteredBlock("carrots");
             potatoes = getRegisteredBlock("potatoes");
             wooden_button = getRegisteredBlock("wooden_button");
-            skull = (BlockSkull)getRegisteredBlock("skull");
+            skull = (BlockSkull) getRegisteredBlock("skull");
             anvil = getRegisteredBlock("anvil");
             trapped_chest = getRegisteredBlock("trapped_chest");
             light_weighted_pressure_plate = getRegisteredBlock("light_weighted_pressure_plate");
             heavy_weighted_pressure_plate = getRegisteredBlock("heavy_weighted_pressure_plate");
-            unpowered_comparator = (BlockRedstoneComparator)getRegisteredBlock("unpowered_comparator");
-            powered_comparator = (BlockRedstoneComparator)getRegisteredBlock("powered_comparator");
-            daylight_detector = (BlockDaylightDetector)getRegisteredBlock("daylight_detector");
-            daylight_detector_inverted = (BlockDaylightDetector)getRegisteredBlock("daylight_detector_inverted");
+            unpowered_comparator = (BlockRedstoneComparator) getRegisteredBlock("unpowered_comparator");
+            powered_comparator = (BlockRedstoneComparator) getRegisteredBlock("powered_comparator");
+            daylight_detector = (BlockDaylightDetector) getRegisteredBlock("daylight_detector");
+            daylight_detector_inverted = (BlockDaylightDetector) getRegisteredBlock("daylight_detector_inverted");
             redstone_block = getRegisteredBlock("redstone_block");
             quartz_ore = getRegisteredBlock("quartz_ore");
-            hopper = (BlockHopper)getRegisteredBlock("hopper");
+            hopper = (BlockHopper) getRegisteredBlock("hopper");
             quartz_block = getRegisteredBlock("quartz_block");
             quartz_stairs = getRegisteredBlock("quartz_stairs");
             activator_rail = getRegisteredBlock("activator_rail");
@@ -436,19 +436,19 @@
             acacia_stairs = getRegisteredBlock("acacia_stairs");
             dark_oak_stairs = getRegisteredBlock("dark_oak_stairs");
             slime_block = getRegisteredBlock("slime");
-            double_plant = (BlockDoublePlant)getRegisteredBlock("double_plant");
-            stained_glass = (BlockStainedGlass)getRegisteredBlock("stained_glass");
-            stained_glass_pane = (BlockStainedGlassPane)getRegisteredBlock("stained_glass_pane");
+            double_plant = (BlockDoublePlant) getRegisteredBlock("double_plant");
+            stained_glass = (BlockStainedGlass) getRegisteredBlock("stained_glass");
+            stained_glass_pane = (BlockStainedGlassPane) getRegisteredBlock("stained_glass_pane");
             prismarine = getRegisteredBlock("prismarine");
             sea_lantern = getRegisteredBlock("sea_lantern");
             standing_banner = getRegisteredBlock("standing_banner");
             wall_banner = getRegisteredBlock("wall_banner");
             red_sandstone = getRegisteredBlock("red_sandstone");
             red_sandstone_stairs = getRegisteredBlock("red_sandstone_stairs");
-            double_stone_slab2 = (BlockSlab)getRegisteredBlock("double_stone_slab2");
-            stone_slab2 = (BlockSlab)getRegisteredBlock("stone_slab2");
+            double_stone_slab2 = (BlockSlab) getRegisteredBlock("double_stone_slab2");
+            stone_slab2 = (BlockSlab) getRegisteredBlock("stone_slab2");
         }
     }
-}
+}
